ceph.spec.in

```diff
@@ -484,12 +484,16 @@ Requires:       python%{_python_buildid}-cherrypy
 Requires:       python%{_python_buildid}-jwt
 Requires:       python%{_python_buildid}-routes
 Requires:       python%{_python_buildid}-werkzeug
+%if 0%{?weak_deps}
```

Comment:
probably we need to backport 147531515b51d7b571b3a4a97e7cbdce239a049a as well, for this macro.

Reply:
Done! Thanks!